### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by ACGaming**
[Merry Christmas!](https://i.pinimg.com/originals/d8/e8/b6/d8e8b64382c85499d2bfc62187fc334e.jpg)

### CODE REVIEW

**Comment by Rayawa** [comment]
Gooooood I'll ask @teddyxlandlee for further infos about this translation, thank you for collaborating to Sweet Potato Mod!

File: src/main/resources/assets/sweet_potato/lang/de_de.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  "block.sweet_potato.enchanted_beetroots": "Verzauberte Rote Bete",
+
+  "subtitles.block.sweet_potato.agroforestry_table.finish": "Nutzpflanze aufgewertet",
+
+  "advancement.husbandry.sweet_potato.get_sweet_potato.title": "Süß und weich",
+  "advancement.husbandry.sweet_potato.get_sweet_potato.description": "Süßkartoffel erhalten",
+  "advancement.nether.sweet_potato.get_magic_cube.title": "Magie der Seele",
+  "advancement.nether.sweet_potato.get_magic_cube.description": "Zauberwürfel erhalten",
+
+  "container.sweet_potato.magic_cube": "Zauberwürfel",
```
Comment (Rayawa) [inline src/main/resources/assets/sweet_potato/lang/de_de.json:55]: Magic Cube's translation has been changed to Magischer Topf.

**Review by Rayawa — COMMENTED** [review]
Accepted. Thank you for collaborating with Sweet Potato Mod.
-----------------------
Akzeptiert. Vielen Dank für die Zusammenarbeit mit Sweet Potato Mod.

File: src/main/resources/assets/sweet_potato/lang/de_de.json (hunk 1 above)
Comment (ACGaming) [reply src/main/resources/assets/sweet_potato/lang/de_de.json:55]: Alright!

**Pull request merged**